src/browser/setupWorker/start/utils/createMessageChannel.ts

```diff
@@ -27,13 +27,6 @@ export class WorkerChannel {
     ...rest: WorkerChannelEventsMap[Event]
   ): void {
     const [data, transfer] = rest
-    this.port.postMessage(
-      { type: event, data },
-      {
-        // @ts-ignore ReadableStream can be transferred
```

Comment:
typescript supports this post versions 4.8, and we _build_ with a newer version that that

Reply:
What about 4.7 that we still support?

Reply:
This is internal to msw, which is _built_ with more modern typescript. 

Because of this, we don't need to ignore it in our build process.  That ignore won't propagate to consumers anyway.

Consumers can consume this either way, as is evident by the compat tests succeeding.